app/Http/Livewire/DetailsComponent.php

```diff
@@ -29,7 +30,8 @@ public function render()
     {
         $product = Product::where('slug', $this->slug)->first();
         $images = Images::where('product_id', $product->id)->get();
+        $reviews = Review::where('product_id', $product->id)->get();
```

Comment:
```suggestion
        $images = $product->images;
        $reviews = $product->reviews;
```
dùng relationship đi em